about.html

```diff
@@ -0,0 +1,124 @@
+          Pas year Irreechaa Celebration Festival
+Take a look at the last two Irreechaa Celebration Festival which took place in Finfinnee and Bishooftuu
+
+          <p class="description-text-2">
+            it brings millions of Oromos from all over Oromia and non-Oromo visitors from other parts of Ethiopia to the shores of Hora (Lake) Arsadi and, as of 2019, to Hora Finfinee in the capital. 
+          </p>
+        </span>
+        <button class="join-btn"> <div class="logo"><a href="index.html">
+          <img src="./assets/images/titleicon.png" alt="logo-icon" class="linklogo-image" /></a></div></button>
+      </section>
+      <hr>
+      <section class="main-program">
+        <h3 class="main-program-title">Past year Irreechaa Celebration Festival</h3>
+        <span class="description-text">
+          Take a look at the last two Irreechaa Celebration Festival which took place in Finfinnee and Bishooftuu.
+        </span>
+      </section>
+    </main>
+    <section class="history-show-image">
+       <div class="image-and-text">
+        <p class="year">2019</p>
+        <p class="year-description">Celebration At Hora Finfinnee</p>
+        <img src="./assets/images/festival/fest1.jpg" alt="group-image-1">
+       </div>  
+       <div class="image-and-text">
+        <p class="year">2020</p>
+        <p class="year-description">Celebrating At Hora Har-Sadee</p>
+        <img src="./assets/images/festival/fest2.png" alt="group-image-1">
+       </div>     
+    </section>
+    <footer class="footer" id="foo">
+      <section class="footer-section">
+      <h3 class="footer-title">Partner</h3>
+      <ul>
+        <li class="contributer-images"><img class="mozilla" src="./assets/images/partner/mozilla.png" alt="mozilla"></li>
+        <li class="contributer-images"><img class="google" src="./assets/images/partner/google.png" alt="google"></li>
+        <li class="contributer-images"><img class="naver" src="./assets/images/partner/naver.png" alt="naver"></li>
+        <li class="contributer-images"><img class="cisco" src="./assets/images/partner/airbnb.png" alt="cisco"></li>
+        <li class="contributer-images"><img class="airbnb" src="./assets/images/partner/microverse.png" alt="airbnb"></li>
+      </ul>
+    </section>
+      <div class="logo-and-description">
+        <div class="logo"><a href="about.html">
+          <img src="./assets/images/titleicon.png" class="linklogo-image"></a></div>
+        <div class="description">
+          <p>Irreechaa  Love,Peace and Unity</p>
+          <p id="desktop">Irreechaa is usually celebrated during the weekend following the Meskel holiday. The festival is observed in many areas of Oromia, but, at the national level, it brings millions of Oromos from all over Oromia and non-Oromo visitors from other parts of Ethiopia to the shores of Hora (Lake) Arsadi and, as of 2019, to Hora Finfinee in the capital. </p>
+        </div>
+      </div>
+      </footer>
```

Comment:
- [ ] your bottom footer in the mobile design has less height than the bottom footer is in the behance design so kindly consider fixing it.
- [ ] Also consider adding the margin-left to the logo in the bottom footer in the mobile design so that it would be similar  to the original design.
your design
![lk](https://user-images.githubusercontent.com/96683843/226133550-02513cc6-f2fa-4c1c-be4a-d343a80a29a6.jpg)
behance design
![mn](https://user-images.githubusercontent.com/96683843/226133573-d4edc6dc-f3ff-4bd9-a2ec-6043bc27bd7f.jpg)